application/src/main/java/org/opentripplanner/apis/transmodel/model/plan/TripPatternType.java

```diff
@@ -225,6 +226,20 @@ public static GraphQLObjectType create(
           .dataFetcher(env -> TripPlanTimePenaltyDto.of(itinerary(env)))
           .build()
       )
+      .field(
+        GraphQLFieldDefinition.newFieldDefinition()
+          .name("emission")
+          .description(
+            """
+            The total emission per preson. The total emission is only available if all transit
```

Comment:
```suggestion
            The total emission per person. The total emission is only available if all transit
```

Reply:
Does "per person" mean that it takes the average number of passengers for this trip and therefore emptier busses have worse emissions?

Reply:
It depends on the feed - input == output. We currently does not have data on it, so we used max_capasity.